STORM_JSTORM_FEATURE_DIFF.md

```diff
@@ -1,30 +1,30 @@
- feature | Storm-0.11.0-SNAPSHOT (and expected pull requests) | JStorm-2.1.0 | Notes | JIRA |
-----------|-----------------------|--------------|-------|----|
- Security | Basic Authentication/Authorization | N/A | The migration is not high risk. But JStorm daeons/connections need to be evaluated
-Scheduler | Resource aware scheduling	(Work In Progress) | <ol><li>Make the tasks of same component distributed evenly across the nodes in cluster.<li>Make number of tasks in a worker is balanced.<li>Try to assign two tasks which are transferring message directly into a same worker to reduce network cost.<li>Support user defined assignment and  to use the result of last assignment	Different solution between Storm and JStorm.</ol> Actually, JStorm has ever used cpu/mem/disk/network demond for scheduling. But we faced the problem of low utilization of cluster. For example, because the configuration of topologys and resources of hardwares in a cluster are different. After running a period, it is possible that no topology will be assigned to the node with enough other resources but just lacking one resource. | Scheduler interface is pluggable so we should be able to support boht schedulers for the time being |

```

Comment:
^ I changed "ever" to "always".   I wasn't sure if it was "always or "never" that the original author meant.  I'm also not entirely sure what I wrote makes sense.  Would appreciate some eyeballs on that.